src/asciidoc/docs/customizations/media.adoc

```diff
@@ -0,0 +1,220 @@
+        "fileOptions": {}
+    }
+}
+----
+
+=== acceptOnly
+
+`acceptOnly` allows you to specify the VCC types that are allowed.
+The value for `acceptOnly` is an array of one or more VCC types.
+If the array contains more than one type, then the user can choose any file of a type that matches one of the types in the array.
+For example, `"acceptOnly": ["image", "video"]` allows the user to select an image file or a video file.
+If the value of `acceptOnly` is an empty array, or if `acceptOnly` is not included in `typeOptions`, then any file type will be allowed.
+
+The possible types are as follows.
+
+* *image* - If you set `acceptOnly` to an array containing `image`, the user will be prompted to choose an image file for the VCC.
+Using a `media` VCC with `acceptOnly` set to `["image"]` is equivalent to using an <<image#, image VCC>>.
+
+* *video* - If you set `acceptOnly` to an array containing `video`, the user will be prompted to choose a video file for the VCC.
+Using a `media` VCC with `acceptOnly` set to `["video"]` is equivalent to using a <<video#, video VCC>>.
+
+* *audio* - If you set `acceptOnly` to an array containing `audio`, the user will be prompted to choose a sound file for the VCC.
+Using a `media` VCC with `acceptOnly` set to `["audio"]` is equivalent to using a <<sound#, sound VCC>>.
+
+* *file* - If you set `acceptOnly` to an array containing `file`, the user will be prompted to choose a file for  the VCC.
+The file selected can be of any type.
+Using a `media` VCC with `acceptOnly` set to `["file"]` is equivalent to using a <<file#, file VCC>>.
+
+=== hideExtensions
+
+The `media` VCC type enables you to hide all asset packs and VCC extensions in cases where they don't make sense (for example, templates for selling premium assets).
+To use this feature in your templates, enable the `hideExtensions` type option.
+In the following code sample, extensions are hidden, so *Browse Image Asset Packs* will not appear as a selection.
+
+[source,json]
+----
+{
+    "key": "hideExtensions",
+    "name": "Image or Video No Extensions",
+    "type": "media",
+    "typeOptions": {
+        "acceptOnly": ["image", "video"],
+        "hideExtensions": true,
+        "videoOptions": {
+            "hls": true
+        }
+    }
+}
+----
+
+The above sample code will display the following selections.
+
+image::mediaNoExtensionsVCC.png[title="Media No Extensions VCC"]
+
+=== videoOptions
+
+For delivery of longer content, HTTP Live Streaming (HLS) can be enabled by setting `hls` to true.
+When enabled, uploaded videos are transcoded for HLS and return an m3u8 playlist.
+Use this feature in conjunction with https://github.com/video-dev/hls.js/[hls.js] for controlling playback.
+
+Pose detection data can be generated by setting `estimatePoses` to true.
+When specified, pose data is available after upload by appending `.poses` to the returned URL.
+For example, `\https://objects.koji-cdn.com/project-id/my-video.mp4.poses`.
+
+=== returnType
```

Comment:
There is way too much unrelated info in the section. We don't need to show full example of how to consume the values; that seems out of scope. We want to say how to set the options and make it clearer what affect they'll have on the custom values collected from the user. For example, this section can be rewritten as follows:

"Specifies the type of the custom value, one of the following:

* `url` – String, URL that points to the file.
* `extended` – (Default) Object with the following structure:
<sample here>"